main.rb

```diff
@@ -0,0 +1,23 @@
+require './Start'
```

Comment:
- [ ] You have a little typo error here, the file name you are importing is `start` not `Start` just the capital letter `S` :+1: